sphinx/ext/doctest.py

```diff
@@ -23,6 +24,8 @@
 from docutils import nodes
 from docutils.parsers.rst import Directive, directives
 
+from distutils.version import StrictVersion as V
```

Comment:
Why abbrviate? `V` is not clear name to me.

Anyway, it should be imported after importing six.
We usually import libraries in order of python standards, 3rd parties, docutils and sphinx's.